tests/src/python/test_qgsmaplayer.py

```diff
@@ -144,6 +145,21 @@ def test_vertical_crs(self):
         self.assertEqual(len(spy), 2)
         self.assertFalse(layer.verticalCrs().isValid())
 
+        # vertical crs info should be present in layer expression context scope
```

Comment:
Maybe better to have this in the [testqgsexpressioncontext.cpp](https://github.com/qgis/QGIS/blob/ad8c0aaac2d39dccf6b9d46b2169ec2307b4a9d4/tests/src/core/testqgsexpressioncontext.cpp#L828) instead?

Reply:
I've moved it to test_qgsexpression.py instead, there's other map layer scope tests in there already